packages/cli/src/cli.js

```diff
@@ -16,6 +16,11 @@ const cli = meow(
   Start options
     --cwd        - Working directory to search patternplate.config.js in
     --port       - Port to start patternplate server on, defaults to 1337, $PORT
+    --listen     - Listen adress to start patternplate server on, defaults to localhost, $LISTEN
+    --privateKeyFname - Filename of the PrivateKey(pem) $PRIVATEKEYFNAME
+    --certChainFname  - Filename of the CertChain(pem) $CERTCHAINFNAME
+    --httpsRedirectPort - If set serve http to redirect to port, needed force https, $HTTPSREDIRECTPORT
```

Comment:
Please change to `--http-port`

Reply:
To keep this symmetrical there should by `--https-port`, too? Given `--*-port`, `--port` should be ignored/disallowed.